### ISSUE TITLE
Shape layer repeater and trim paths bug

### ISSUE THREAD

**Issue by Joolyan** [airbnb/lottie-web#2401]
**Tell us about your environment**
macOS 10.13.6

**Browser and Browser Version:**
Chrome Version 87.0.4280.88 (Official Build) (x86_64)
Firefox 83.0 (64-bit)

**After Effects Version:**
Version 17.5.1

Reproduce using the attached .aep or .json files and lottie-web v5.7.5 on macOS 10.13.6 with all tested browsers.

The shape layer contains two groups, the first contains a path with a repeater, the second a rounded rectangle with two trim paths. The repeater in one group appears to repeat the path in the next group.

The work round is to change the order of the groups, so the group with the trim layers appears before the group with the repeater, or remove the repeater and duplicate the paths manually.

Hope this helps identify the issue.

[Demo.zip](https://github.com/airbnb/lottie-web/files/5719429/Demo.zip)

**Comment by bodymovin** [comment]
hi, version 5.7.6 should have fixed this

**Comment by Joolyan** [comment]
Thanks for looking into this issue.

Unfortunately I've tested the new 5.7.6 release using the same .aep project file and the problem remains the same.

I've attached two images which might help illustrate the problem. NB. The repeater seems to repeat the trim paths property.

As this issue has been closed, I'm not sure if this requires a new issue?

Thanks for your help.
![With_Bug](https://user-images.githubusercontent.com/19827502/104911840-62511180-5983-11eb-9861-bc48d34372b1.png)
![Without_Bug](https://user-images.githubusercontent.com/19827502/104911853-6715c580-5983-11eb-93e6-e248ca4ba6f8.png)
